src/pages/coc.re

```diff
@@ -16,14 +17,43 @@ let make = (_children) => {
         )
       </p>
       <h2> (ste("Need Help?")) </h2>
-      <p> (ste("You have our contact details in the emails we’ve sent.")) </p>
+      <p>
+        <div> (ste("Reason Team")) </div>
+        <div>
+          (ste("Twitter "))
+          <a href="https://twitter.com/reasonconf" target="_blank">
+            (ste("@reasonconf"))
+          </a>
+        </div>
+        <div>
+          (ste("Email "))
+          <a href="mailto:hi@reason-conf.com"> (ste("hi@reason-conf.com")) </a>
+        </div>
+      </p>
+      <p>
+        <div> (ste("Niklaus Graf")) </div>
+        <div>
+          (ste("Twitter "))
+          <a href="https://twitter.com/nikgraf" target="_blank">
+            (ste("@nikgraf"))
+          </a>
+        </div>
+        <div>
+          (ste("Email "))
+          <a href="mailto:nik@reason-conf.com">
+            (ste("nik@reason-conf.com"))
+          </a>
+        </div>
+      </p>
       <h2> (ste("The Quick Version")) </h2>
       <p>
         (
           ate([|
-            (ste("Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from the conference ")),
-            <em>(ste("without a refund"))</em>,
-            (ste(" at the discretion of the conference organisers.")),
+            ste(
```

Comment:
Consider using Markdown for this instead. See https://github.com/ReasonVienna/reason-conf/blob/master/src/pages/sponsors.re#L40

Reply:
Can we do this in another PR? This was just reformatting from rfmt

Reply:
I refactored it